clang/lib/Frontend/HeaderIncludeGen.cpp

```diff
@@ -112,11 +112,17 @@ class HeaderIncludesJSONCallback : public PPCallbacks {
 /// an array of separate entries, one for each non-system source file used in
 /// the compilation showing only the direct includes and imports from that file.
 class HeaderIncludesDirectPerFileCallback : public PPCallbacks {
+  struct HeaderIncludeInfo {
+    SourceLocation location;
```

Comment:
```suggestion
    SourceLocation Location;
```
LLVM uses PascalCase for variable and member names. Ditto for the rest of the patch.